extensions/amp-accordion/amp-accordion.md

```diff
@@ -267,6 +267,18 @@ it collapses all sections of the accordion. To specify a section, add the
 </button>
 ```
 
+### Actions in Bento Mode
```

Comment:
What about listening for Bento component events? Is there a specific syntax for that?

Reply:
Here is an example of how to listen for Bento component events.

```
<amp-accordion id="eventsAccordion" animate='true'>
  <section id="section1">
    <h2>Section 1</h2>
    <div>Puppies are cute.</div>
  </section>
  <section id="section2">
    <h2>Section 2</h2>
    <div>Kittens are furry.</div>
  </section>
  <section id="section3">
    <h2>Section 3</h2>
    <div>Elephants have great memory.</div>
  </section>
</amp-accordion>

<script>
  (async () => {
    const accordion = document.querySelector('#eventsAccordion');
    await customElements.whenDefined('amp-accordion');
    const api = await accordion.getApi();

    // when section 1 expands, section 2 also expands
    // when section 2 collapses, section 1 also collapses
    const section1 = document.querySelector('#section1');
    const section2 = document.querySelector('#section2');
    section1.addEventListener('expand', () => api.expand('section2'));
    section2.addEventListener('collapse', () => api.collapse('section1'));
  })();
</script>
```

Reply:
For `accordion` specifically, I discovered a bug when creating this example, so this won't work until the bug fix is merged.  But the syntax is correct and can be used as an example.

Here's the bug link to track status: https://github.com/ampproject/amphtml/pull/32141